settings.gradle.kts

```diff
@@ -1 +1,35 @@
 rootProject.name = "rewrite-launchdarkly"
+
+include("plugin")
+include("metrics")
```

Comment:
@timtebeek, looks like you copied this from the Gradle plugin, so these two statements are not necessary to this project.

Reply:
Thanks! Fixed here and elsewhere. :)